eolearn/core/eoexecution.py

```diff
@@ -181,11 +180,7 @@ def run(self, workers: int | None = 1, multiprocess: bool = True, **tqdm_kwargs:
         if self.save_logs:
             self.filesystem.makedirs(self.report_folder, recreate=True)
 
-        log_paths: Sequence[str | None]
-        if self.save_logs:
-            log_paths = self.get_log_paths(full_path=False)
-        else:
-            log_paths = [None] * len(self.execution_kwargs)
+        log_paths = self.get_log_paths(full_path=False) if self.save_logs else it.repeat(None)
```

Comment:
would it make sense to provide the `times` parameter to `repeat`, since we know it?

Reply:
any reason for it?

Reply:
infinities are scary! 

my thinking was that it's better to fail if potentially a larger number is requested, rather than passing silently

🤷  

but not so important, I guess

Reply:
tried adding it to make you feel safer, but the line gets too long and more difficult to read :(

Reply:
I will accidentally click resolve and look away while you merge